src/shared/clients/ecsClient.ts

```diff
@@ -84,12 +84,28 @@ export class DefaultEcsClient {
             return listTasksResponse.taskArns ?? []
         } catch (error) {
             getLogger().error(
-                `ecs: Failed to get tasks for Cluster "${cluster}" and Service "${serviceName}": ${error} `
+                `ecs: Failed to get tasks for Cluster "${cluster}" and Service "${serviceName}": ${error}`
             )
             throw error
         }
     }
 
+    public async updateService(cluster: string, serviceName: string, enable: boolean): Promise<void> {
+        const sdkClient = await this.createSdkClient()
+
+        const params: ECS.UpdateServiceRequest = {
+            cluster,
+            service: serviceName,
+            enableExecuteCommand: enable,
+            forceNewDeployment: true,
+        }
+        try {
+            sdkClient.updateService(params).promise()
```

Comment:
This isn't being awaited so it'll never throw, instead it will be reported as an unhandled promise